tests/integration/test_ucsc_radiometer_agent_integration.py

```diff
@@ -0,0 +1,49 @@
+import time
+from datetime import datetime
+
+import ocs
+import pytest
+from flask import jsonify, request
+from http_server_mock import HttpServerMock
+from integration.util import create_crossbar_fixture
+from ocs.base import OpCode
+from ocs.testing import create_agent_runner_fixture, create_client_fixture
+
+pytest_plugins = "docker_compose"
+
+wait_for_crossbar = create_crossbar_fixture()
+run_agent = create_agent_runner_fixture(
+    "../socs/agents/ucsc_radiometer/agent.py",
+    "radiometer",
+    args=["--log-dir", "./logs/"],
+)
+client = create_client_fixture("pwvs")
+
+
+@pytest.fixture
+def http_mock():
+    app = HttpServerMock(__name__)
+
+    @app.route("/", methods=["GET"])
+    def route_fn():
+        if request.method == "GET":
+            time_now = datetime.now()
+            timestamp = time.mktime(time_now.timetuple())
+            data = {'pwv': 1.2, 'timestamp': timestamp}
+            return jsonify(data)
+        else:
+            assert False, "Bad query"
+
+    return app.run("127.0.0.1", 5002)
+
+
+@pytest.mark.integtest
+def test_ucsc_radiometer_acq(wait_for_crossbar, http_mock, run_agent, client):
+    with http_mock:
+        resp = client.acq.stop()
+        resp = client.acq.start(test_mode=True)
+        resp = client.acq.wait()
+        print(resp)
+        assert resp.status == ocs.OK
+        print(resp.session)
+        assert resp.session['op_code'] == OpCode.SUCCEEDED.value
```

Comment:
This test takes a whole minute for me to run locally. (Took 1:09 and 1:10 the two times I ran it.) I think this is just due to the acq process waiting a whole minute to query things. The two solutions I can think of are:
1. Mock out the sleep function to return immediately. This is done in several tests already, though I'm not sure if it's done for the pacemaker...
2. Have the process skip the wait when in 'test_mode'.

The actual test part of this should be <1 second. There will be overhead for starting/stopping the crossbar server and Agent, but definitely a minute is too long.

Reply:
Hmm.. I thought step 2 would be more straightforward to try but I can't get that to work. I took it to mean that if you're in 'test_mode', have the `acq` process in the agent skip any kind of pacemaker rate-limit stuff. I don't end up seeing a solution that way. Did you have a different idea? Meanwhile, I'll take a look at other tests for step 1